examples/05_advanced/plot_haxby_mass_univariate.py

```diff
@@ -122,6 +122,8 @@
 
 # Use the fmri mean image as a surrogate of anatomical data
 from nilearn import image
+from nilearn.image import get_data
```

Comment:
Maybe better to use image.get_data in this example, as image is already imported